src/utils/Api.js

```diff
@@ -0,0 +1,125 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    (this._baseUrl = baseUrl), (this._headers = headers);
+  }
+
+  getAppInfo() {
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else {
+          Promise.reject(`Error ${res.status}`);
+        }
+      })
+      .catch(console.error);
```

Comment:
`      .catch(console.error);
`

You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js`  after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.